src/traces/scatter/hover.js

```diff
@@ -72,7 +72,9 @@ module.exports = function hoverPoints(pointData, xval, yval, hovermode) {
                 yLabelVal: di.y
             });
 
-            if(di.tx) pointData.text = di.tx;
+            if(di.htx) pointData.text = di.htx;
+            else if(trace.hovertext) pointData.text = trace.hovertext;
+            else if(di.tx) pointData.text = di.tx;
```

Comment:
This is what I came up with:

- If trace `hoverinfo` contains a `'text'` flag and `hovertext` is not set the `text` elements will be seen in the hover labels
- If trace `hoverinfo` contains a `'text'` flag and `hovertext` is set, `hovertext` takes precedence over `text` i.e. the `hoverinfo` element will be seen in the hover labels